flink-libraries/flink-table/src/test/scala/org/apache/flink/table/api/scala/stream/table/validation/GroupWindowAggregationsTest.scala

```diff
@@ -16,19 +16,19 @@
  * limitations under the License.
  */
 
-package org.apache.flink.table.api.scala.stream.table
+package org.apache.flink.table.api.scala.stream.table.validation
```

Comment:
I think tests in `validation` packages should all test for a `ValidationException`. Can you split this class?